<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MBP.local" failures="0" tests="5" name="ExtenteReportMethods.UsingListeners" time="19.024" errors="1" timestamp="2025-02-15T14:16:31 IST" skipped="1">
  <testcase classname="ExtenteReportMethods.UsingListeners" name="main2" time="4.196">
    <system-out>
      <![CDATA[onTestStart for - main2
Main2 executed.
onTestSuccess for - main2]]>
    </system-out>
  </testcase> <!-- main2 -->
  <testcase classname="ExtenteReportMethods.UsingListeners" name="main3" time="4.911">
    <system-out>
      <![CDATA[onTestStart for - main3
Main3 executed.
onTestSuccess for - main3]]>
    </system-out>
  </testcase> <!-- main3 -->
  <testcase classname="ExtenteReportMethods.UsingListeners" name="main1" time="4.650">
    <system-out>
      <![CDATA[onTestStart for - main1
Main1 executed.
onTestSuccess for - main1]]>
    </system-out>
  </testcase> <!-- main1 -->
  <testcase classname="ExtenteReportMethods.UsingListeners" name="main5" time="4.252">
    <skipped/>
    <system-out>
      <![CDATA[onTestSkipped for - main5]]>
    </system-out>
  </testcase> <!-- main5 -->
  <testcase classname="ExtenteReportMethods.UsingListeners" name="main4" time="1.015">
    <error message="Method ExtenteReportMethods.UsingListeners.main4() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method ExtenteReportMethods.UsingListeners.main4() didn't finish within the time-out 1000
at java.base/java.lang.Thread.sleepNanos0(Native Method)
at java.base/java.lang.Thread.sleepNanos(Thread.java:496)
at java.base/java.lang.Thread.sleep(Thread.java:527)
at ExtenteReportMethods.UsingListeners.main4(UsingListeners.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
    </error>
    <system-out>
      <![CDATA[onTestStart for - main4
Main4 executed.
onTestFailure for - main4]]>
    </system-out>
  </testcase> <!-- main4 -->
  <system-out>
    <![CDATA[onStart
onTestStart for - main1
Main1 executed.
onTestSuccess for - main1
onTestStart for - main2
Main2 executed.
onTestSuccess for - main2
onTestStart for - main3
Main3 executed.
onTestSuccess for - main3
onTestStart for - main4
Main4 executed.
onTestFailure for - main4
onTestStart for - main5
onTestSkipped for - main5]]>
  </system-out>
</testsuite> <!-- ExtenteReportMethods.UsingListeners -->
